cli/chaosli/cmd/root.go

```diff
@@ -22,6 +24,8 @@ import (
 	"github.com/spf13/viper"
 )
 
+var VERSION string
+var API_LIB_PATH string = fmt.Sprintf("chaosli-api-lib/v1beta1/%v", VERSION)
 var cfgFile string
```

Comment:
Is it expected to have variable names which do not respect naming standards (I'd have expected `Version` and `APILibPath` or `ApiLibPath` here?

Reply:
I'll conform to the naming standards then

Reply:
Done, variable is now APILibPath